python/tk_framework_desktopserver/shotgun/constants.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2017 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+import os
+
+# RPC return codes.
+SUCCESSFUL_LOOKUP = 0
+UNSUPPORTED_ENTITY_TYPE = 2
+CACHING_ERROR = 3
+COMMAND_SUCCEEDED = 0
+COMMAND_FAILED = 1
+
+# Pipeline configurations.
+OVERRIDE_CONFIG_PATH = os.environ.get("TK_BOOTSTRAP_CONFIG_OVERRIDE")
+
+if OVERRIDE_CONFIG_PATH is not None:
```

Comment:
Yes. The override environment variable will be used for the manager's base_configuration, but won't be allowed to override PipelineConfiguration entities. This logic here establishes what that base_configuration will be.